File: src/main/java/com/optica/services/search/SearchJobOfferService.java
package com.optica.services.search;

import java.util.List;

import com.optica.domain.JobOffer;

public interface SearchJobOfferService {

	List<JobOffer> findJobOffersByKeyword(String keyword);
	
	List<JobOffer> findJobOffersByCity(String city);
	
	List<JobOffer> findJobOffersByKeywordAndCity(String keyword,String city);
	
}

File: src/main/kotlin/com/optica/services/search/SearchJobOfferService.kt
package com.optica.services.search

import com.optica.domain.JobOffer

interface SearchJobOfferService {

    fun findJobOffersByKeyword(keyword: String): List<JobOffer>

    fun findJobOffersByCity(city: String): List<JobOffer>

    fun findJobOffersByKeywordAndCity(keyword: String, city: String): List<JobOffer>

}
